hat/assets/js/apps/Iaso/domains/completenessStats/hooks/useCompletenessStatsColumns.tsx

```diff
@@ -204,6 +211,18 @@ export const useCompletenessStatsColumns = (
             accessor: 'actions',
             sortable: false,
             Cell: settings => {
+                const formStats = settings.row.original.form_stats;
+                const orgunitId = settings.row.original.org_unit.id;
+                let hasFormSubmissions = false;
+                Object.entries(formStats).forEach(
+                    // eslint-disable-next-line no-unused-vars, @typescript-eslint/no-unused-vars
+                    ([_key, value]: [string, Record<string, any>]) => {
+                        if (value.itself_has_instances > 0) {
+                            hasFormSubmissions = true;
+                        }
+                    },
+                );
```

Comment:
using valuesIn from lodash you can make something like that:
`const hasFormSubmissions = valuesIn(formStats).some(stat => stat.itself_has_instances > 0 );`